source/base/qprojector.cc

```diff
@@ -285,8 +310,9 @@ QProjector<2>::project_to_subface(const ReferenceCell::Type reference_cell_type,
                                   std::vector<Point<2>> &   q_points,
                                   const RefinementCase<1> &)
 {
-  Assert(reference_cell_type == ReferenceCell::Type::Quad, ExcNotImplemented());
-  (void)reference_cell_type;
+  Assert(reference_cell_type == ReferenceCell::Type::Tri ||
+           reference_cell_type == ReferenceCell::Type::Quad,
+         ExcNotImplemented());
```

Comment:
@peterrum -- This is not needed as well I assume.

Reply:
Yes.